src/main/java/net/sf/jabref/logic/groups/ExplicitGroup.java

```diff
@@ -168,10 +168,6 @@ public String getTypeId() {
         return ExplicitGroup.ID;
     }
 
-    public int getNumEntries() {
```

Comment:
Does the removal of this method introduce some performance issues? Or is iterating over all entries still fast enough?